scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/FileSystemService.java

```diff
@@ -146,4 +145,37 @@ public void createFileInDirectory(String username, String type, String id, Strin
         }
     }
 
+    @Override
+    public void createZipFile(String username, String type, String id, String version)
```

Comment:
This method should return something (e.g. the path to the created zip file). Otherwise, if we change the implementation later to put zip files somewhere else for example, anyone who calls this method has to change as well, which is very errrorprone.